packages/AsyncLoader/src/AsyncLoader.unit.mjs

```diff
@@ -218,3 +221,26 @@ test('dispatches fail event if loading content failed', async(t) => {
     t.is(errors.length, 0);
 });
 
+test('loads data from endpoint url passed in event payload', async(t) => {
+    const { document, window, errors } = await setup(true);
+    window.fetch = polyfillFetch(200, '<h2>Test</h2>');
+    const loader = createElement(document,
+        `<async-loader
+            data-event-endpoint-property-name="endPointUrl"
+            data-trigger-event-name="loadData"
+        >
+            <div data-content-container>Initial</div>
+            <template data-loading-template>Loading ...</template>
+            <template data-error-template>Error: {{message}}</template>
+        </async-loader>`);
+    document.body.appendChild(loader);
+    const container = loader.querySelector('[data-content-container]');
+    t.is(container.innerHTML, 'Initial');
+
+    loader.dispatchEvent(new window.CustomEvent('loadData', { bubbles: true, detail: { endPointUrl: 'testContent.html' } }));
+    t.is(container.innerHTML, 'Loading ...');
+
+    await new Promise((resolve) => setTimeout(resolve, 0));
+    t.is(container.innerHTML, '<h2>Test</h2>');
+    t.is(errors.length, 0);
+});
```

Comment:
Gern noch einen Test, in dem beide Parameter vorhanden sind.

Reply:
Bringt dieser Test so wirklich was? Die Response kommt ja durch die Zeile: `window.fetch = polyfillFetch(200, '<h2>Test</h2>');` . Dann kann gar nicht beobachtet werden, ob eine andere Url gefetcht wird.

Reply:
Schau dir mal die `polyfillFetch`-Funktion genauer an.

Reply:
Ok, ich glaube ich habe rausgefunden wies geht